csrankings-x.csv

```diff
@@ -37,6 +37,7 @@ Xianfang Sun,Cardiff University,https://www.cardiff.ac.uk/people/view/118165-sun
 Xianfeng David Gu,Stony Brook University,http://www3.cs.stonybrook.edu/~gu,Y063_CIAAAAJ
 Xianfeng Gu,Stony Brook University,http://www3.cs.stonybrook.edu/~gu,Y063_CIAAAAJ
 Xiang Ao 0001,Chinese Academy of Sciences,https://aoxaustin.github.io,W8wrWfMAAAAJ
+Xiang 'Anthony' Chen,Univ. of California - Los Angeles,https://hci.prof,I2W13z0AAAAJ

```

Comment:
Needs entry for Xiang Anthony Chen as well.

Reply:
Done, thanks.